cbo/terraform-solution-implementation/01-create-role-in-vsphere/versions.tf

```diff
@@ -6,8 +6,8 @@ terraform {
   required_providers {
     vsphere = {
       source  = "hashicorp/vsphere"
-      version = ">= 2.2.0"
+      version = ">= 2.0.2"
     }
   }
-  required_version = ">= 1.2.0"
+  required_version = ">= 1.0.0"
```

Comment:
Any reason for not using >= 1.2.0 for Terraform. I believe that is what we have for CBA.

Reply:
> **Note**
> We need to update the same for PCA.